ProteGO/Common/Application/AppDelegate.swift

```diff
@@ -38,6 +38,7 @@ final class AppDelegate: UIResponder, UIApplicationDelegate  {
     func applicationWillEnterForeground(_ application: UIApplication) {
         self.advertiser?.setMode(.enabledAllTime)
         self.scanner?.setMode(.enabledAllTime)
+        self.resolver.resolve(DangerStatusManagerType.self)?.updateCurrentDangerStatus()
```

Comment:
Może lepiej byłoby przetrzymać to jako wartość opcjonalną w AppDelegate, żeby za każdym wejściem do appki nie musiała być wyciągana z resolvera?

Reply:
póki co to ten manager jest wykorzysywany w appdelegate tylko w tym jednym miejscu, więc nie widze potrzeby aby go na siłę jeszcze przetrzymywać tutaj w appdelegate.

IMO niepotrzebnie zaciemni kod, a nic nie wniesie. 

To wyciaganie z DI nie jest na tyle "ciężką" operacją abyśmy musieli ją jakoś znacząco optymalizować IMO